### DISCUSSION TITLE
ImportError: cannot import name 'setup_default_warnings'

### DISCUSSION THREAD

**Discussion by Tortoise17** [explosion/spaCy#8650] [Help: Installation] [answered]
I want to report you for this current version released yesterday 3.1.0

```
from spacy import load

  File "/home/usr/anaconda3/envs/zml/lib/python3.6/site-packages/spacy/__init__.py", line 6, in <module>
    from .errors import setup_default_warnings

ImportError: cannot import name 'setup_default_warnings'
```
Please have a look if you can.

## Your Environment

* Operating System:
* Python Version Used:
* spaCy Version Used:
* Environment Information:

**Comment by adrianeboyd** [comment, marked as answer]
Hi, that looks like it might be a problem with your install. Can you try uninstalling spacy completely and reinstalling? If it still doesn't work, try it in a new venv to test?

It looks like you're using conda, so if you were using `pip`, you can try installing spacy with `conda install -c conda-forge spacy` instead.
